test/e2e/tags.go

```diff
@@ -0,0 +1,45 @@
+package e2e
+
+// The use of ginkgo tags is encouraged for describing tests.
+// This file defines test tags and annotations.
+// Each annotation should have a corresponding description.
+
+type Tag string
+
+const (
+	// May fail in secured clusters which aren't specifically opened up to allow certain operations.
```

Comment:
can you give an example of a test this would be applied to?

`[NonStandardSecurity]` seems kinda verbose. wonder if there's a better word we can use...